stimpacks/aws/login.go

```diff
@@ -1,36 +1,67 @@
 package aws
 
 import (
-	"github.com/skratchdot/open-golang/open"
-
-	"encoding/json"
 	"errors"
 	"fmt"
-	"io/ioutil"
-	"net/http"
-	"net/url"
-	"strings"
 	"time"
+
+	stimaws "github.com/PremiereGlobal/stim/pkg/aws"
+	"github.com/skratchdot/open-golang/open"
 )
 
 // TODO: Move this to a global config
 var stimURL = "https://github.com/PremiereGlobal/stim"
 
+// Profile This is our custom profile we'll use to keep track of things
+type Profile struct {
```

Comment:
feel like this should be defined in the pkg not the stimpack

Reply:
Added this to the pkg in a more generic form and I added the option of passing additional fields to append to the ini section.